### ISSUE TITLE
Multiple Blockquotes separated by a newline being grouped in single blockquote

### ISSUE THREAD

**Issue by Tooa** [secure-77/Perlite#27]
**Describe the bug**

Parsedown does not identify different blockquotes when the blockquotes are just separated by a newline. The following markdown file:

```md
> This is a blockquote

> This is another blockquote
```

results in the following HTML

```html
<blockquote>
<p>This is a blockquote</p>
<p>This is another blockquote</p>
<blockquote>
```

Note that this is an upstream issue.

**To Reproduce**

* Go to the [Parsedown Demo](https://parsedown.org/demo)
* Enter the markdown file listed in the issue description

**Expected behavior**

```html
<blockquote>
<p>This is a blockquote</p>
</blockquote>
<blockquote>
<p>This is another blockquote</p>
</blockquote>
```

**Workaround**

* Add a newline and regular text between the elements

**Screenshots**

![image](https://user-images.githubusercontent.com/5375334/176273306-6123d44f-cae2-4f80-83f2-7273021dfd1b.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04
 - Browser Firefox 102

**Solution proposal**

I tested the newest Parsedown [beta v2.0.0-beta-1](https://github.com/erusev/parsedown/releases/tag/v2.0.0-beta-1) and it looks like the problem is fixed in this version. 

We may consider switching to this beta release and test how stable it works with our existing test documents. When upgrading to the latest version, I suggest to use Php Composer as well for managing the dependency.

Here is a step by step guide, how I tested the behavior with the beta release:

**Setup**
```bash
$ sudo apt install php8.1
$ php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
$ php composer-setup.php
$ php -r "unlink('composer-setup.php');"
# Dependencies
$ sudo apt-get install php8.1-mbstring
$ php ../composer.phar require erusev/parsedown:v2.0.0-beta-1
$ php demo.php
```

**demo.php**
```php
<?php

require __DIR__ . '/vendor/autoload.php';

use Erusev\Parsedown\Configurables\Breaks;
use Erusev\Parsedown\Configurables\SafeMode;
use Erusev\Parsedown\Configurables\StrictMode;
use Erusev\Parsedown\State;
use Erusev\Parsedown\Parsedown;


$markdown = <<<EOD
> This is a blockquote

> this is another blockquote
EOD;


$state = new State([
    new Breaks(true),
    new SafeMode(true),
    new StrictMode(false)
]);

$Parsedown = new Parsedown($state);
echo $Parsedown->toHtml($markdown);
?>
```

![image](https://user-images.githubusercontent.com/5375334/176274430-d4a607b9-b465-4f69-9357-11ad9389d4c9.png)

**Comment by secure-77** [comment]
Hi @Tooa,

I can confirm the bug. I also recognized that there is a new beta release for parsedown, but for the moment I don`t want to switch to the beta release. The effort for testing and possible error handling is not worth it. At least not because of this "little" bug. 

I will wait until there is a stable 2.0 release or if we really "need" to switch for other reasons. Iam completely with you regarding the composer, also thought about it and I want to do that for the JS dependencies as well (as soon I have some time for).

**Comment by secure-77** [comment]
fixed in the next release 1.5.1
